application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -57,7 +57,7 @@ public Schedule createSchedule() {
     @Override
     public void runRoutine(JDA jda) {
         Instant now = Instant.now();
-        database.write(context -> context.selectFrom(PENDING_REMINDERS)
+        database.write(any -> any.selectFrom(PENDING_REMINDERS)
```

Comment:
this was a mistake. `any` is supposed to indicate **i dont care what or who this parameter is, since i never use it**. but here, you are using it.

Reply:
but does it matter? isn't it supposed to be just a parameter being passed into the the function? be it `context` or `any`?

Reply:
but the reader needs to understand what it is, if u are using it. it is the database-context.

i mean, its clear that its important to have a good variable naming. `dogs` instead of `foos` or `objects`. but when you do not actually use it, and the parameter is just forced upon you - but you are not interested, then its idiomatic to call it `any` (or in other languages `_`).

for example, if you have a `List<Pair<String, Int>> persons` int kotlin and want to print all the strings, then u would write `persons.forEach { (name, _) -> println(name) }`. See how it is important to clearly name what the left parameter is, since we didnt use a well named class `Person` but `Pair` instead. But we dont care what the second thing is (`age`), so naming it would only distract the reader. Thats kinda the idea.

Reply:
from what i can understand, think it's safe to  conclude if im gonna use something, i should care about the name of variable. Thanks for the explanation.

Reply:
"when i read any, i know that it is never used inside"